.github/workflows/codeql.yml

```diff
@@ -25,8 +25,22 @@ concurrency:
   cancel-in-progress: ${{ github.event_name != 'push' }}
 
 jobs:
+  pre-checks:
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.dependabot.outputs.result }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: dependabot check
```

Comment:
We don't want to run codeql if the PR is from a fork or a dependabot PR (whatever label) so it would be good to call check-ext-build on pull_request event too on this workflow.
And on push, I don't see the point of calling dependabot-check (but I agree it does not hurt)

Reply:
ok let's merge what it does not hurt and add raise a task for the fork